src/components/about/About.js

```diff
@@ -70,6 +73,11 @@ const styles = StyleSheet.create({
   aboutDescription: {
     maxWidth: normalize(270),
   },
+  imageContainer: {
+    flex: 1,
+    flexDirection: 'column',
+    maxHeight: '50%',
```

Comment:
can you use percentages on react-native?

Reply:
Yes you can, but if you prefer I change it for numbers